version.go

```diff
@@ -0,0 +1,3 @@
+package customerio
+
+const Version = "3.1.0"
```

Comment:
3.2.0 when you release!

Reply:
🤦‍♀️ Updated to 3.2. Thanks for catching that!